site/src/components/card/Card.tsx

```diff
@@ -0,0 +1,82 @@
+import { ComponentPropsWithoutRef, useMemo, useRef } from "react";
+import clsx from "clsx";
+import Link from "@docusaurus/Link";
+import PencilIcon from "@site/static/img/pencil.svg";
+import CodeIcon from "@site/static/img/code.svg";
+import ArrowsIcon from "@site/static/img/arrows.svg";
+import useOnScreen from "../../utils/useOnScreen";
+
+import styles from "./Card.module.css";
+
+type CardIconName = "pencil" | "code" | "arrows";
+
+type CardIconsType = {
+  name: CardIconName;
+  icon: JSX.Element;
+}[];
+
+const cardIcons: CardIconsType = [
```

Comment:
Not a blocker, but perhaps we should open this up in future PRs. Perhaps we could update the props so that you can pass in any CSS color value (which would include things like `var(--uitk-color-...)`) for the keyline and any React component for the image part.

That might make the card more re-usable in other areas of the website